cmd/ef_tests/levm/runner/levm_runner.rs

```diff
@@ -224,7 +233,20 @@ pub fn get_state_transitions(execution_report: &TransactionReport) -> Vec<Accoun
         let code = if account.info.bytecode.is_empty() {
             None
         } else {
-            Some(account.info.bytecode.clone())
+            let potential_new_bytecode_hash = code_hash(&account.info.bytecode);
+            let current_bytecode = current_db
+                .get_account_code(potential_new_bytecode_hash)
+                .expect("Error getting account code by hash");
+            let code = account.info.bytecode.clone();
```

Comment:
```suggestion
            let new_bytecode = account.info.bytecode.clone();
```
I don't know exactly why but the block of code inside the `else {` of `let code` was kinda confusing to read. I din't understand it's purpose on the first place but now I get it.

Reply:
I can add some comments explaining what is happening there.